### PR TITLE
L6: the secret-life of objects

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

File: src/br/js/L6-the-secret-life-of-objects/A_vector_type.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Vec {
+    constructor(numA, numB) {
+        this.numA = numA;
```
Comment (carin91) [inline src/br/js/L6-the-secret-life-of-objects/A_vector_type.js:3]: assign the names of the Euclidean plane {x: numA, y: numB}

File: src/br/js/L6-the-secret-life-of-objects/A_vector_type.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class Vec {
+    constructor(numA, numB) {
+        this.numA = numA;
+        this.numB = numB;
+    }
+
+    plus(other) {
+        return new Vec(this.numA + other.numA, this.numB + other.numB);
```
Comment (carin91) [inline src/br/js/L6-the-secret-life-of-objects/A_vector_type.js:8]: refactor it considering the new names of the keys.

File: src/br/js/L6-the-secret-life-of-objects/A_vector_type.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    constructor(numA, numB) {
+        this.numA = numA;
+        this.numB = numB;
+    }
+
+    plus(other) {
+        return new Vec(this.numA + other.numA, this.numB + other.numB);
+    }
+
+    minus(other) {
```
Comment (carin91) [inline src/br/js/L6-the-secret-life-of-objects/A_vector_type.js:11]: refactor it considering the new names of the keys.

**Review by carin91 — APPROVED** [review]

File: src/br/js/L6-the-secret-life-of-objects/A_vector_type.js (hunk 1 above)
Comment (Brandonz07) [reply src/br/js/L6-the-secret-life-of-objects/A_vector_type.js]: I changed the names

**Review by carin91 — APPROVED** [review]

**Pull request merged**